bower.json

```diff
@@ -33,8 +33,16 @@
     "lodash": "~3.10.1",
     "components-font-awesome": "~4.5.0",
     "angular-poller": "~0.4.5",
-    "moment": "~2.11.2",
-    "pusher": "^3.0.0"
+    "moment": "^2.11.2",
+    "angular-toastr": "^1.7.0",
+    "pusher": "^3.0.0",
+    "angular-translate": "^2.11.0",
+    "angular-translate-handler-log": "^2.11.0",
+    "angular-translate-loader-static-files": "^2.11.0",
+    "angular-translate-storage-cookie": "^2.11.0",
```

Comment:
@alexnoox I'm not 100% sure, the strategy currently seems to be managed in impac-angular.config.coffee. Can you please take a look at this file, should that be happening there? @clemthenem added that I think and I'm not too sure what these strategy's are doing.